deployability/modules/jobflow/tests/test_dag.py

```diff
@@ -6,12 +6,12 @@
 from unittest.mock import patch, MagicMock, call
 import pytest
 
-from workflow_engine.workflow_processor import DAG
+from jobflow.workflow_processor import DAG
```

Comment:
Shouldn't the name of the workflow_processor method and WorkflowFilw be changed?

Reply:
Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8